dsgrid/registry/dimension_mapping_registry_manager.py

```diff
@@ -147,8 +146,52 @@ def validate_records(self, config: DimensionMappingsConfig, warn_only=False):
         Check:
         - from_id and to_id column names
         - check for duplicate IDs and log a warning if they exist (sometimes we want them to exist if it is an aggregation)
+
+        Implemented:
+        - allow duplicates in from_id if fraction col is explicitly filled out, log warning when fraction sums > 1;
+        - duplicates in to_id is always allowed but logged when found.
```

Comment:
I don't think expected conditions should warn. What happened to adding some meta-data to the DimensionMappingConfig so we know what to expect?